cmd/dep/integration_test.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2017 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+
+	"github.com/golang/dep/test"
+)
+
+func TestIntegration(t *testing.T) {
+	test.NeedsExternalNetwork(t)
+	test.NeedsGit(t)
+
+	filepath.Walk("testdata", func(path string, info os.FileInfo, err error) error {
```

Comment:
Fair enough, but yeah, I think the extra separator is still worthwhile. Given that we're defining semantics around filesystem organization, I think it's better to have the explicit top-level choice.